dev-lua/luarocks/luarocks-3.2.1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A deployment and management system for Lua modules"
+HOMEPAGE="http://www.luarocks.org"
+SRC_URI="http://luarocks.org/releases/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="test"
+
+RESTRICT="!test? ( test )"
+
+BDEPEND="virtual/pkgconfig"
+DEPEND="dev-lang/lua:=
+		dev-libs/openssl:=
+		net-misc/curl"
+RDEPEND="${DEPEND}"
+
+src_configure() {
+	# econf doesn't work b/c it passes variables the custom configure can't
+	# handle
+	./configure \
+			--prefix="${EPRIFIX}/usr" \
+			--with-lua-lib="${EPRIFIX}/usr/$(get_libdir)" \
+			--rocks-tree="${EPRIFIX}/usr/$(get_libdir)/lua/luarocks" \
+			|| die "configure failed"
+}
+
+src_test() {
+	busted ./spec || die
+}
+
+src_install() {
+	default
+	{ find "${D}" -type f -print0 | xargs --null sed -i -e "s:${D}::g"; } || die "sed failed"
```

Comment:
`-exec` find option can be used here instead of xargs

Reply:
fixed